packages/e2e-tests/plugins/interactive-blocks/directive-bind/view.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array( 'dependencies' => array( '@wordpress/interactivity' ) );
```

Comment:
@anton-vlasenko, how we can disable the file name check for the asset files? The same patter is used in WordPress core without any issues.

Reply:
I added an exclude here which seems correct:

```xml
	<rule ref="WordPress.Files.FileName.NotHyphenatedLowercase">
		<exclude-pattern>/phpunit/*</exclude-pattern>
		<exclude-pattern>*.asset.php</exclude-pattern>
	</rule>
```

It looks like Core ignores asset files entirely, but disabling just that rule on them is good enough here.

https://github.com/WordPress/wordpress-develop/blob/9a616a573434b432a5efd4de21039250658371fe/phpcs.xml.dist#L69C1-L70C1

Reply:
I see I added one of the lines 5 years ago:

```
<exclude-pattern>/src/wp-includes/assets/*</exclude-pattern>
```

😅